tests/testdata/graph.json

```diff
@@ -1 +1,2020 @@
\ No newline at end of file
+[
```

Comment:
I would include the generated file without prettifying it.

Reply:
👍